.github/workflows/container_description.yml

```diff
@@ -14,6 +14,7 @@ jobs:
   PushDockerHubReadme:
     runs-on: ubuntu-latest
     name: Push README to Docker Hub
+    if: github.repository == 'prometheus/prometheus'
```

Comment:
Would this work?

```suggestion
    if: secrets.DOCKER_HUB_LOGIN != ''
```

Reply:
(we could also try `github.repository_owner == 'prometheus'` if we know all those repos live there)

Reply:
We also use `prometheus-community`.

I think the env secret checks are better if they work. Since those are directly required by the script.

Reply:
Using `secrets` isn't supported in `jobs.<name>.if`. Only 
`github`, `needs`, `vars` and `inputs` are available according to the docs: https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability:~:text=jobs.%3Cjob_id%3E.if,cancelled%2C%20success%2C%20failure

I'll update this PR to use `github.repository_owner == 'prometheus' || github.repository_owner == 'prometheus-community'`.

Reply:
Boo, that's unfortunate.